plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -67,14 +83,25 @@ function isBlank(str: string) {
  */
 export class ElasticSearchSearchEngine implements SearchEngine {
   private readonly elasticSearchClient: Client;
+  private readonly highlightOptions: ElasticSearchHighlightOptions;
 
   constructor(
     private readonly elasticSearchClientOptions: ElasticSearchClientOptions,
     private readonly aliasPostfix: string,
     private readonly indexPrefix: string,
     private readonly logger: Logger,
+    highlightOptions?: ElasticSearchHighlightOptions,
```

Comment:
UUID thing makes total sense.  Main concern now is for folks who provide custom translators.  In `ElasticSearchSearchEngine::query()`, the result item highlights' pre/post tags are set based on `this.highlightOptions`.  Would a custom translator have access to the UUID value set there?

Reply:
Ah I didn't not consider custom translators, thanks for pointing that out! I refactored the interface a bit so that translators are passed the highlight options so it will be available for any custom translator implementations.

Reply:
Nice!  Thank you.